src/Test.rsc

```diff
@@ -17,3 +18,39 @@ void testMain(loc input) {
   TEnv tenv = collect(ast);
   set[Message] msgs = check(ast, tenv, g.useDef);
 }
+
+Value getValue(value val) {
+  switch (val) {
+    case int n:
+      return vint(n);
+    case bool b:
+      return vbool(b);
+    case str s:
+      return vstr(s);
+    default:
+      throw "Unsupported value <val>";
+  }
+}
+
+void testEval() {
+  // modify these ones for testing
+  loc location = |cwd:///examples/test1.myql|;
+  map[str, value] venv_map = ("hasSoldHouse": true, "sellingPrice": 300, "privateDebt": 20);
```

Comment:
Shouldn't this be created by the initialEnv function?

Reply:
This was before compiling the forms was available. The venv_map is filled with variables and their values for test purposes, e.g. to see how the forms respond to certain user input.

Reply:
Ok. Ignore my comment then.